custom/app-setup/kafka/chaos-exporter/service-account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: litmus
+  namespace: litmus
```

Comment:
What is the use of this SA - Do we have role and rolebinding associated with it?

Reply:
can we apply this https://raw.githubusercontent.com/litmuschaos/chaos-operator/master/deploy/rbac.yaml instead?